static/js/admin/vue-permissions-editor/src/components/patronRoles.vue

```diff
@@ -512,6 +516,7 @@
              * See mixins/displayModal.js
              */
             showModal() {
+                this.unsaved_changes = this.hasUnsavedChanges;
```

Comment:
I don't see any usages of `this.unsaved_changes`, maybe it can be removed?

Reply:
It's used by the modal mixin to determine how it'll close